ext/tidy/config.m4

```diff
@@ -58,8 +58,10 @@ if test "$PHP_TIDY" != "no"; then
   PHP_ADD_INCLUDE([$TIDY_INCDIR])
 
   dnl Add -Wno-ignored-qualifiers as this is an issue upstream
```

Comment:
Is this still a problem upstream? Unfortunately 1393834 that added this doesn't seem to reference anything...

Reply:
Yes, still happening with tidy-html5 versions before 5.7.20:

```
/usr/include/tidy/tidy.h:2095:13: warning: type qualifiers ignored on function return type [-Wignored-qualifiers]
 2095 | TIDY_EXPORT const ctmbstr TIDY_CALL TidyLangWindowsName( const tidyLocaleMapItem *item );
      |             ^~~~~
/usr/include/tidy/tidy.h:2101:13: warning: type qualifiers ignored on function return type [-Wignored-qualifiers]
 2101 | TIDY_EXPORT const ctmbstr TIDY_CALL TidyLangPosixName( const tidyLocaleMapItem *item );
      |             ^~~~~
```

https://github.com/htacg/tidy-html5/issues/866